workers/ohsome_quality_analyst/api.py

```diff
@@ -152,14 +177,21 @@ async def _fetch_report(
     url: str,
     bpolys: Optional[str] = None,
     dataset: Optional[str] = None,
-    feature_id: Optional[Union[int]] = None,
+    feature_id: Union[int, str, None] = None,
     fid_field: Optional[str] = None,
 ):
     if bpolys is not None:
         bpolys = await load_bpolys(bpolys)
 
+    if feature_id:
+        feature_id = int_or_str_param_type(feature_id)
+
     report = await oqt.create_report(
-        name, bpolys=bpolys, dataset=dataset, feature_id=feature_id, fid_field=fid_field
+        name,
+        feature=bpolys,
```

Comment:
the name `feature` can be confusing, because `bpolys` might contain a `Feature` or a `Geometry`

Reply:
No, thats not quite right. The API parameter `bpolys` can be of type `Feature` or `Geometry`, but after parsing it using `load_bpolys()` it is always a `Feature` object.

Reply:
I renamed the variable `bpolys` to `feature` after `load_bpolys()` has been called.